src/Serialization/Fields.jl

```diff
@@ -532,6 +533,27 @@ function load_object(s::DeserializerState, ::Type{<:EmbeddedNumFieldElem}, paren
   return parent_field(loaded_alg_elem)
 end
 
+################################################################################
+# QQBar
+
+@register_serialization_type QQBarField
+@register_serialization_type QQBarFieldElem
+
+function save_object(s::SerializerState, q::QQBarFieldElem)
+  save_data_dict(s) do
+    save_object(s, minpoly(q), :minpoly)
+    save_object(s, AcbField()(q), :acb )
+  end
+end
+
+function load_object(s::DeserializerState, ::Type{QQBarFieldElem})
+  Qx, x = QQ[:x]
+  CC = AcbField()
+  min_poly = load_object(s, PolyRingElem{QQ}, Qx, :minpoly)
+  approximation = load_object(s, AcbFieldElem, CC, :acb)
+  return filter(x -> overlaps(approximation, CC(x)), roots(QQBarField(), min_poly))[1]
```

Comment:
One should add a check that there is only one thing overlapping (or use `only`). But if this fails, then the original object cannot be recovered. I will push some suggestion to fix this.

Reply:
Yes/no. One can ask for as much precision as one wants during the saving. Sou you would look (during the saving) at `approximation AcbField(prec)(q)` and increase the prec until the filter dance returns a unique element. Do you know what I mean? If not, I can push the suggestion.

Reply:
Ya I know what you mean. I'll push a change sometime tomorrow